src/components/InsightCard/Pulse.svelte

```diff
@@ -47,6 +71,10 @@
 
   .profile {
     --img-size: 24px;
+
+    :global(body:not(.desktop)) & {
+      --img-size: 40px;
+    }
   }
 
   .text {
```

Comment:
These changes are redundant due to [task description](https://www.notion.so/santiment/Mobile-fixes-Iter-1-Part-2-d7ee33392add4ae99e2f1167b79252ea#1a77347a971247859f394e6ca2a1e0f3). Just render regular `InsightCard` on mobile for pulse